langchain4j-jlama/src/main/java/dev/langchain4j/model/jlama/JlamaStreamingChatModel.java

```diff
@@ -55,7 +70,7 @@ public JlamaStreamingChatModel(Path modelCachePath,
             loader = loader.workingDirectory(workingDirectory);
 
         this.model = loader.load();
-        this.temperature = temperature == null ? 0.7f : temperature;
+        this.temperature = temperature == null ? 0.3f : temperature;
```

Comment:
Is there a default value for `temperature` in Jlama? Perharps we should not have defaults on LC4j side?

Reply:
Models don't have defaults like this, I could add a default on jlamas side, but in my api its required.

Reply:
OK let's keep it then, thanks